### ISSUE TITLE
use an env variable for the docker-compose file

### ISSUE THREAD

**Issue by Grummfy** [jenssegers/captain#3]
Hi,
I think we should have a way to use a .env, a config file in /etc or a variable from the environment to look at the docker-compose file. Perhaps could add the support for multiple one (find dir1 dir2 ...) ?

**Comment by jenssegers** [comment]
Yeah I'm thinking of create a `~/.captain/config` file that allows some customization :)

**Comment by Grummfy** [comment]
seems good to me
perhaps like docker (for advanced version) with a captain config --set home path to home

**Comment by Grummfy** [comment]
for info : https://golang.org/pkg/os/#UserConfigDir or https://github.com/shibukawa/configdir if it's not working properly

**Comment by territory-sschneider** [comment]
`direnv` is also not working with captain unfortunately
